shm/cargo.hpp

```diff
@@ -4,8 +4,9 @@
 
 class Cargo {
 public:
-    Cargo& operator+=(size_t amount);
-    Cargo& operator-=(size_t amount);
+    Cargo& operator+=(size_t);
+    Cargo& operator-=(size_t);
+    bool operator==(const Cargo&);
```

Comment:
I prefer types with names of variables in hpp. I think that it will be more clear that we want increase amount.
But don't know which way is preferred by our group.

Reply:
Ok ok I will maybe add that :)